package/obs/rmt-server.spec

```diff
@@ -22,10 +22,14 @@
 %define conf_dir     %{_sysconfdir}/rmt
 %define rmt_user     _rmt
 %define rmt_group    nginx
-%define ruby_version %{rb_default_ruby_suffix}
+
+# Only build for the distribution default Ruby version
+%define rb_build_versions     %{rb_default_ruby}
+%define rb_build_ruby_abis    %{rb_default_ruby_abi}
+%define ruby_version          %{rb_default_ruby_suffix}
 
 Name:           rmt-server
-Version:        2.5.8
+Version:        2.5.9
```

Comment:
#569 has already claimed this version number, and we've actually deployed 2.5.9 to our servers. :thinking:

Reply:
Bumped to 2.5.10 on 2fb5e5c.